src/errorcodes-net-cli/errorcodes-net-cli.csproj

```diff
@@ -20,7 +20,12 @@
       <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic.Workspaces" Version="4.6.0" />
       <PackageReference Include="Microsoft.CodeAnalysis.Workspaces.MSBuild" Version="4.6.0" />
       <PackageReference Include="System.CommandLine" Version="2.0.0-beta4.22272.1" />
-      <PackageReference Include="YamlDotNet" Version="15.1.2" PrivateAssets="all" GeneratePathProperty="true" />
+      <PackageReference Include="YamlDotNet" Version="15.1.2" />
```

Comment:
The `PrivateAssets="all"` made the package not be included with the tool. Remove that so it works properly.